exalearn-api/ExaLearn.WebApi/appsettings.json

```diff
@@ -8,7 +8,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DbContext": "Server=localhost;Port=5432;Username=postgres;Password=;Database=exalern_db",
+    "DbContext": "Server=localhost;Password=123456;Port=5432;Username=postgres;Password=123;Database=exalern_db",
     "HostingContext": "linkToFilesHosting"
```

Comment:
revert your private changes for DbContext

Reply:
+